polyfills/Element/prototype/dataset/detect.js

```diff
@@ -0,0 +1,6 @@
+(document.documentElement.dataset &&
+	(
+		Object.getOwnPropertyDescriptor(Element.prototype, 'dataset')  &&
+		Object.getOwnPropertyDescriptor(Element.prototype, 'dataset').get
```

Comment:
Sorry for being slow... what does that correlation tell us?  I'm concerned about browsers that have neither getOwnPropertyDescriptor nor dataset.  The detect for dataset will error in browsers that don't have getOwnPropertyDescriptor, unless a polyfill for getOwnPropertyDescriptor is loaded first.  It will be most of the time, but I am concerned that there are edge cases, eg. RUM, where it is not.